doc/source/index.rst

```diff
@@ -140,11 +168,12 @@ parameters. For example:
      'asr', 'reciprocal', ...
      'use_c', true);
 
-To see all the available optional parameters, try:
+To see all the available optional parameters, try one of:
 
 .. code-block:: matlab
 
-  help(euphonic.CoherentCrystal)
+  help euphonic.CoherentCrystal
+  doc euphonic.CoherentCrystal
```

Comment:
Below, line 194/195 should be amended to say 'help on the `horace_disp` function can be seen with `help(euphonic.CoherentCrystal.horace_disp)`'